config/lib/seed.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+  chalk = require('chalk'),
+  crypto = require('crypto'),
+  User = mongoose.model('User');
+
+console.log(chalk.bold.red('Warning:  Database seeding is turned on'));
+
+//If production only seed admin if it does not exist
+if (process.env.NODE_ENV === 'production') {
```

Comment:
Basically it will seed the admin account in production only if it does not already exist....  In DEV and TEST it always seeds the admin and user accounts so you always have something to login with.  I am just trying to get something in so first time users don't have to use mongo to insert an admin user.  Once its seeded they can use the admin console to make any updates that they want to make.